composites/HelpCenter/views/VideoTutorial.js

```diff
@@ -15,7 +15,7 @@ const VideoTutorialContainer = styled.div`
 `;
 
 const VideoContainer = styled.div`
-	float: left;
+	${ props => props.hasParagraphs ? "float: left;" : "margin: auto;" }
```

Comment:
`margin: auto` is also taking the top and bottom margin, is that intended? I think `margin: 0 auto` would be a better solution. Perhaps `display:block` might be needed as well, because of cross-browser.